mandatory/2-function-creation.js

```diff
@@ -15,27 +23,64 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  if (totalPrice >= 200) {
+    return totalPrice * 0.9;
+  }
+  else {
+    return totalPrice * 0.95;
 
-/*
+  }
+
+
+
+}
+
+
+/*let limit = 0;
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+
+  for (let i = 0; i <= limit; i++) {
+    if (i % 2 === 1) {
+      console.log(i);
+    }
+  }
+
+
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
+function buyTwoGetTheCheapestFree(price1, price2) {
+  if (price1 > price2) {
+    return price1;
+  }
+  else {
+    return price2;
 
+  }
+
+
+}
```

Comment:
Ensure you do not leave lots of unnecessary whitespace in functions.

Also, the else block should not be on a newline.